docs/docs/overview/quickstart/get-started-with-kotlin.md

```diff
@@ -0,0 +1,288 @@
+Before running the code, make sure you understand the following:
+- The value of `payload` is an array of the data that you want to be dynamically injected into the notification template content.
+- The value of `subscriberId` is the ID of the subscriber on Novu. Replace `12345` with your subscriber ID.
+
+Run the code to trigger a notification! :e-mail:
+
+## Topics
+
+Novu provides a simple API that offers an easy interface for triggering notifications to multiple subscribers at once. This API is called **Topics** and allows users to manage their bulk notifications without having to implement complex loops. A topic is identified by a custom key that is provided by the user, and this key will be the identifier used in the Topics API.
+
+::::info
+The topic key should be unique and can't be changed once chosen. Novu also caters for key uniqueness behind the scenes.
+::::
+
+A topic can have multiple subscribers who will receive a notification whenever a message is sent to the topic.
+
+### Create a Topic
+
+Copy and paste the following code into your app to create a topic:
+```kotlin
+suspend fun createTopic(): Any? {
+        val novu = Novu(apiKey = "API_KEY")
+        val createTopicRequest = CreateTopicRequest(
+            key = "key",
+            name = "name"
+        )
+        return CoroutineScope(Dispatchers.IO).async {
+            novu.createTopic(createTopicRequest)
+        }.await()
+    }
+```
+Before running the code, make sure you understand the following:
+
+- When creating a `key`, ensure it is unique and accurately identifies the topic. Document naming conventions and communicate them to team members to avoid confusion and ensure a smooth workflow.
+- The value of `name` should be a descriptive topic name.
+
+### Add subscribers to a Topic
+
+Copy and paste the following code into your app to add subscribers a topic:
+```kotlin
+suspend fun addSubscriberToTopic(): Any? {
+        val novu = Novu(apiKey = "API_KEY")
+        val topicKey = "key"
+        val requestBody = SubscriberList(listOf("name"))
+        return CoroutineScope(Dispatchers.IO).async {
+            novu.addSubscribers(topicKey, requestBody)
+        }.await()
+    }
+```
+On the other hand, if you want to remove subscribers from a topic, do the following:
+```kotlin
+suspend fun removeSubscriberFromTopic(): Any? {
+        val novu = Novu(apiKey = "API_KEY")
+        val topicKey = "key"
+        val requestBody = SubscriberList(listOf("name"))
+        return CoroutineScope(Dispatchers.IO).async {
+            novu.removeSubscriber(topicKey, requestBody)
+        }.await()
+    }
+```
+
+### Sending a Notification to a Topic
+
+Thanks to the topics feature, it is possible to trigger a notification to all subscribers assigned to a topic. This helps avoid listing all subscriber identifiers in the `to` field of the notification trigger.
+
+:::info
+This feature is not yet available on the Kotlin SDK, this doc will be updated once it is released on Kotlin.
```

Comment:
I'm trying to get us to make a new release. Messaged the initial Kotlin maintainer though

Reply:
Sorry, I am not sure I got the action point. What are you suggesting I do?

Reply:
Don't do anything. It's action on my end.